plugin/trino-redshift/README.md

```diff
@@ -18,3 +18,123 @@ test.redshift.jdbc.password=<password>
+create ephemeral Amazon Redshift clusters: 
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "PassRoleToRedshiftCluster",
+            "Effect": "Allow",
+            "Action": "iam:PassRole",
+            "Resource": "arn:aws:iam::894365193301:role/redshift-ci"
+        },
+        {
+            "Sid": "RedshiftClusterManagement",
+            "Effect": "Allow",
+            "Action": [
+                "redshift:DeleteTags",
+                "redshift:DeleteCluster",
+                "redshift:CreateTags",
+                "redshift:CreateCluster",
+                "redshift:DescribeClusters",
+                "redshift:DescribeLoggingStatus"
+            ],
+            "Resource": "arn:aws:redshift:us-east-2:894365193301:cluster:trino-redshift-ci-cluster-*"
+        },
+        {
+            "Sid": "DescribeRedshiftVpcComponents",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeInternetGateways",
+                "ec2:DescribeAddresses",
+                "ec2:DescribeAvailabilityZones",
+                "ec2:DescribeVpcs",
+                "ec2:DescribeAccountAttributes",
+                "ec2:DescribeSubnets",
+                "ec2:DescribeSecurityGroups"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+### AWS S3 setup
+
+The `trino-redshift` tests rely on a Redshift cluster 
+having TPCH tables filled with data.
+Create an AWS S3 bucket and add to it the parquet content 
+of `tpch` tables saved locally through the `trino-hive` connector 
+via commands like:
+
+```
+CREATE TABLE hive.tiny.table_name AS TABLE tpch.sf1.table_name
+```
+
+The content of the S3 bucket should look like this:
+
+```
+s3://<your_bucket>/tpch/tiny/<table_name>/*.parquet
+```
+
+where `table_name` is:
+
+- `customer`
+- `lineitem`
+- `nation`
+- `orders`
+- `part`
+- `partsupp`
+- `region`
+- `supplier`
+
+# Local Development with Redshift Ephemeral Cluster on Trino CI AWS infrastructure
+
+The CI tests spin up a Redshift cluster on-demand for test execution. The Maven tests run first which populate the users and test data
+needed for the product tests to run. The scripts under `trino/bin/redshift` control creating and destroying a cluster which
+the `ci.yml` file uses. They can also be used for local development.
+
+1. Make sure you are authenticated to the AWS account dedicated for Trino CI
+2. Change directory to `bin/redshift` and execute `./bin/redshift/local_setup_aws_redshift.sh`. This will spin up a Redshift cluster via Pulumi scripts.
+    * It takes roughly four minutes for the script to complete
+3. Once finished, you'll see VM arguments for Maven tests printed to the terminal as well as environment variables for product tests.
+   You can use these VM arguments in IntelliJ as well.
+4. **Important!** Once you are done, run `./bin/redshift/local_delete_aws_redshift.sh` to delete the cluster.
```

Comment:
We need scripts for CI, but we don't need the scripts to be used by developers.

Removing expectation that the scripts work for engineers too will allow us to simplify stuff like names, param handling, and will also free us from "local_*" version of the scripts. Less code is better.

For example, CI needs Redshift instance to be publicly accessible and may use a very strong randomized password to access it.
An engineer may use an always-up redshift instance, available over a VPN or SSH tunnel, and fixed password, as that makes it easiest to run tests directly from IntelliJ.